spec/models/spree/shipping_method_spec.rb

```diff
@@ -3,7 +3,12 @@
 module Spree
   describe ShippingMethod do
     it "is valid when built from factory" do
-      expect(create(:shipping_method)).to be_valid
+      expect(
+        build_stubbed(
+          :shipping_method,
+          shipping_categories: [Spree::ShippingCategory.new(name: 'Test')]
+        )
```

Comment:
does build_stubbed run callbacks? In this case, I tend to use `build` for this test cases.

Reply:
No. Just replaced `build_stubbed` with `build`.